views/js/loader/taoQtiTest.min.js

```diff
@@ -1,2 +1,2 @@
x in navMap[partIndex].sections)if(navMap[partIndex].sections.hasOwnProperty(sectionIndex)&&!0===navMap[partIndex].sections[sectionIndex].active){sectionItems=navMap[partIndex].sections[sectionIndex].items;break}return sectionItems},skip:function skip(){var self=this,doSkip=function(){self.disableGui(),self.actionCall("skip")};this.shouldDisplayEndTestWarning()?this.displayEndTestWarning(doSkip):doSkip()},timeout:function timeout(){var self=this;this.disableGui(),this.testContext.isTimeout=!0,this.updateTimer(),this.killItemSession(function(){var confirmBox=$(".timeout-modal-feedback"),testContext=self.testContext,confirmBtn=confirmBox.find(".js-timeout-confirm, .modal-close");testContext.numberCompletedSection===testContext.numberItemsSection?testMetaData.addData({SECTION:{SECTION_EXIT_CODE:testMetaData.SECTION_EXIT_CODE.COMPLETE_TIMEOUT}}):testMetaData.addData({SECTION:{SECTION_EXIT_CODE:testMetaData.SECTION_EXIT_CODE.TIMEOUT}}),self.enableGui(),confirmBox.modal({width:500}),confirmBtn.off("click").on("click",function(){confirmBox.modal("close"),self.actionCall("timeout")})})},setTestContext:function setTestContext(testContext){this.testContext=testContext,this.itemServiceApi=eval(testContext.itemServiceApiCall),this.itemServiceApi.setHasBeenPaused(testContext.hasBeenPaused)},initMetadata:function initMetadata(){testMetaData=testMetaDataFactory({testServiceCallId:this.itemServiceApi.serviceCallId})},getSessionStateService:function getSessionStateService(){return sessionStateService||(sessionStateService=this.testContext.sessionStateService({accuracy:1e3})),sessionStateService},update:function update(testContext){var self=this;$controls.$itemFrame.remove();var $runner=$("#runner");$runner.css("height","auto"),this.getSessionStateService().restart(),this.setTestContext(testContext),this.updateContext(),this.updateProgress(),this.updateNavigation(),this.updateTestReview(),this.updateInformation(),this.updateRubrics(),this.updateTools(testContext),this.updateTimer(),this.updateExitButton(),this.resetCurrentItemState(),this.initMetadata(),$controls.$itemFrame=$("<iframe id=\"qti-item\" frameborder=\"0\" scrollbars=\"no\"/>"),$controls.$itemFrame.appendTo($controls.$contentBox),this.testContext.itemSessionState===this.TEST_ITEM_STATE_INTERACTING&&!1===self.testContext.isTimeout?($doc.off(".testRunner").on("serviceloaded.testRunner",function(){self.afterTransition(),self.adjustFrame(),$controls.$itemFrame.css({visibility:"visible"})}),this.itemServiceApi.loadInto($controls.$itemFrame[0],function(){})):self.afterTransition()},updateInformation:function updateInformation(){!0===this.testContext.isTimeout?feedback().error(__("Time limit reached for item \"%s\".",this.testContext.itemIdentifier)):this.testContext.itemSessionState!==this.TEST_ITEM_STATE_INTERACTING&&feedback().error(__("No more attempts allowed for item \"%s\".",this.testContext.itemIdentifier))},updateTools:function(testContext){var showSkip=!1,showSkipEnd=!1,showNextSection=!!testContext
```

Comment:
I don't think you need to commit the minified files and their maps.

Reply:
Right you are. Will be generated in release

Reply:
Wait, now you deleted them completely. Put them back as they were :D

Reply:
If I delete that, `npx grunt taoqtitestbundle` doesn't compile for me. Can you check it?

Reply:
I don't want you to delete them. Just revert them to their `develop` state.

Reply:
I think it's done. Not sure because strange things happen

Reply:
How can you call this done?
![image](https://user-images.githubusercontent.com/2943256/79456269-7cbe1f00-7fee-11ea-8ca7-d9c3a11e4efe.png)

Reply:
Let me see, because I merge from develop and nothing to do.

Reply:
This is not how git works.
Of course there's nothing to be merged to your branch from `develop`. There's nothing new there. Your branch is based on it.